main.tf

```diff
@@ -37,45 +37,45 @@ resource "google_cloud_scheduler_job" "job" {
  *****************************************/
 
 module "pubsub_topic" {
-  source     = "github.com/terraform-google-modules/terraform-google-pubsub?ref=v0.1.0"
-  topic      = "${var.topic_name}"
-  project_id = "${var.project_id}"
+  source     = "github.com/terraform-google-modules/terraform-google-pubsub?ref=v0.2.0"
```

Comment:
This will need to be updated to source 1.0.0 from the Terraform registry.

Blocked by https://github.com/terraform-google-modules/terraform-google-pubsub/pull/7